src/NUnit3TestAdapterInstallV2/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("NUnit3TestAdapterInstall")]
-[assembly: AssemblyCopyright("")]
+[assembly: AssemblyCopyright("Copyright © 2011-2016, NUnit Software")]
```

Comment:
Actually, I just copied it from the NUnit 2 adapter, but I will update to Charlie Poole for this release.